### PR TITLE
Ran/login app 2

### PR DESCRIPTION
**Pull request by stormcrow85**

### CODE REVIEW

File: demo/loginApp/demo.js [hunk 1]
```diff
@@ -13,12 +13,4 @@ angular.module('demo', ['znk.infra-web-app.loginApp'])
         $rootScope.$on('$translatePartialLoaderStructureChanged', function () {
             $translate.refresh();
         })
-    })
```
Comment (assafshp) [inline demo/loginApp/demo.js]: why did you remove it?

File: dist/loginApp/loginApp.js [hunk 2]
```diff
@@ -125,6 +124,7 @@
 
                     scope.loginSubmit = function(){
                         if (!scope.d.loginFormData) {
+                            window.alert('form is empty!');
```
Comment (assafshp) [inline dist/loginApp/loginApp.js]: $window

File: dist/loginApp/loginApp.js [hunk 3]
```diff
@@ -165,6 +164,7 @@
 
                     scope.signupSubmit = function(){
                         if (!scope.d.signupFormData) {
+                            window.alert('form is empty!');
```
Comment (assafshp) [inline dist/loginApp/loginApp.js]: $

File: dist/loginApp/loginApp.js [hunk 4]
```diff
@@ -649,7 +641,7 @@ angular.module('znk.infra-web-app.loginApp').run(['$templateCache', function($te
     "        toefl: d.appContext === d.availableApps.TOEFL,\n" +
     "    }\">\n" +
     "    <header>\n" +
-    "        <div class=\"logo\"></div>\n" +
+    "        <a class=\"logo\" href=\"http://www.zinkerz.com\"></a>\n" +
```
Comment (assafshp) [inline dist/loginApp/loginApp.js]: http -> //

File: dist/main.js [hunk 5]
```diff
@@ -5263,6 +5262,7 @@ angular.module('znk.infra-web-app.invitation').run(['$templateCache', function($
 
                     scope.loginSubmit = function(){
                         if (!scope.d.loginFormData) {
+                            window.alert('form is empty!');
```
Comment (assafshp) [inline dist/main.js]: $

File: dist/main.js [hunk 6]
```diff
@@ -5303,6 +5302,7 @@ angular.module('znk.infra-web-app.invitation').run(['$templateCache', function($
 
                     scope.signupSubmit = function(){
                         if (!scope.d.signupFormData) {
+                            window.alert('form is empty!');
```
Comment (assafshp) [inline dist/main.js]: $

File: dist/main.js [hunk 7]
```diff
@@ -5787,7 +5779,7 @@ angular.module('znk.infra-web-app.loginApp').run(['$templateCache', function($te
     "        toefl: d.appContext === d.availableApps.TOEFL,\n" +
     "    }\">\n" +
     "    <header>\n" +
-    "        <div class=\"logo\"></div>\n" +
+    "        <a class=\"logo\" href=\"http://www.zinkerz.com\"></a>\n" +
```
Comment (assafshp) [inline dist/main.js]: //

File: src/components/loginApp/directives/signupForm.directive.js [hunk 8]
```diff
@@ -25,6 +24,7 @@
 
                     scope.signupSubmit = function(){
                         if (!scope.d.signupFormData) {
+                            window.alert('form is empty!');
```
Comment (assafshp) [inline src/components/loginApp/directives/signupForm.directive.js]: $

File: src/components/loginApp/templates/loginApp.directive.html [hunk 9]
```diff
@@ -6,7 +6,7 @@
         toefl: d.appContext === d.availableApps.TOEFL,
     }">
     <header>
-        <div class="logo"></div>
+        <a class="logo" href="http://www.zinkerz.com"></a>
```
Comment (assafshp) [inline src/components/loginApp/templates/loginApp.directive.html]: //

File: demo/loginApp/demo.js (hunk 1 above)
Comment (stormcrow85) [inline demo/loginApp/demo.js]: not in use by the demo
